velox/docs/develop/connectors.rst

```diff
@@ -0,0 +1,91 @@
+==========
+Connectors
+==========
+
+Connectors allow to read and write data to and from external sources.
+This concept is similar to `Presto Connectors <https://prestodb.io/docs/current/develop/connectors.html>`_.
+The :ref:`TableScanNode<TableScanNode>` operator reads external data via a connector.
+The :ref:`TableWriteNode<TableWriteNode>` operator writes data externally via a connector.
+The various connector interfaces in Velox are described below.
+
+Connector Interface
+-------------------
+
+.. list-table::
+   :widths: 10 40
+   :header-rows: 1
+
+   * - Interface Name
+     - Description
+   * - ConnectorSplit
+     - Basic unit that describes a chunk of data for processing.
```

Comment:
In my experience, folks do not quite understand what a split is. It may help to provide an example. For example, 

```
A chunk of data to process. For example, a single file.
```